hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -126,7 +139,15 @@ set_ignore_cdc_paths -name tlul_async_fifo_err -through_signal $tlul_async_data
 ## Apply Constraints   ##
 #########################
 
-read_sdc $CONSTRAINT
+#read_sdc $CONSTRAINT
+create_scenario {sdc env}
+current_scenario sdc
+read_sdc  $CONSTRAINT  -output_env constraints.sdc.env
+current_scenario env
+read_env constraints.sdc.env
+touch /workspaces/repo/hw/top_earlgrey/cdc/top_user.env
```

Comment:
we probably can't hardcode paths like these, so it would be good to find the right reference.

Reply:
Fixed